source/event/2017/toronto.md

```diff
@@ -12,7 +12,11 @@ tags: [event ]
 
 ## IIIF Working Meeting - Toronto - 2017
 
-The International Image Interoperability Framework ([IIIF][home-page]) Working Meeting in Toronto provides an in-person venue for active IIIF community participants and [IIIF groups][groups] to make progress on advancing specific aspects of IIIF such as specifications, software, and community development (as opposed to the annual IIIF conference, which is more geared toward sharing knowledge). Attendance is free, but capacity is limited, so [registration][registration] is required in order to ensure that we do not exceed the capacity of the venue. Stay tuned to the [IIIF-Discuss][iiif-discuss] email list for announcements and more information.
+The International Image Interoperability Framework ([IIIF][home-page]) Working Meeting in Toronto provides an in-person venue for active IIIF community participants and [IIIF groups][groups] to make progress on advancing specific aspects of IIIF such as specifications, software, and community development (as opposed to the annual IIIF conference, which is more geared toward sharing knowledge). Attendance is free, but capacity is limited, so [registration][registration] is required in order to ensure that we do not exceed the capacity of the venue.
```

Comment:
Do we need "(as opposed to the annual IIIF conference, which is more geared toward sharing knowledge)"? If so, I'd move it out of parentheses, but that's just me.

Reply:
ok